app/events/decidim/ai/spam_detection/spam_digest_event.rb

```diff
@@ -0,0 +1,127 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Ai
+    module SpamDetection
+      class SpamDigestEvent < Decidim::Events::BaseEvent
+        include Decidim::Events::EmailEvent
+
+        def self.types
+          [:email, :notification]
+        end
+
+        def resource
+          return @resource unless @resource.is_a?(Decidim::Organization)
+
+          OpenStruct.new(organization: @resource)
+        end
+
+        def email_intro
+          org_name =
+            organization.name[I18n.locale.to_s].presence ||
+            organization.name.dig("machine_translations", I18n.locale.to_s).presence ||
+            organization.name["en"].presence ||
+            organization.name.values.compact.first
+
+          I18n.t(
+            "decidim.ai.spam_detection.digest.summary",
+            count: spam_count,
+            frequency_label:,
+            organization: org_name,
+            moderations_url:
+          ).html_safe
```

Comment:
We might prefer a sanitize method but I'm not sure that it's mandatory and we could probably keep the html_safe method